pkg/virt-api/webhooks/validating-webhook/admitters/vmi-create-admitter.go

```diff
@@ -1764,6 +1764,18 @@ func validateDisks(field *k8sfield.Path, disks []v1.Disk) []metav1.StatusCause {
 
 				}
 			}
+
+			// Reject defining DedicatedIOThread to a disk with SATA bus since this configuration
+			// is not supported in libvirt.
+			isIOThreadsWithSataBus := disk.DedicatedIOThread != nil && *disk.DedicatedIOThread &&
+				(disk.DiskDevice.Disk != nil) && strings.EqualFold(disk.DiskDevice.Disk.Bus, "sata")
+			if isIOThreadsWithSataBus {
+				causes = append(causes, metav1.StatusCause{
+					Type:    metav1.CauseTypeFieldValueNotSupported,
+					Message: fmt.Sprintf("Disk with DedicatedIOThread and a SATA bus is not supported"),
```

Comment:
How would you feel about changing the error message to something like "IOThreads are not supported for disks on a SATA bus." ?

Reply:
Indeed your message is clearer. I'll change that. Thanks! :)